public/video-ui/src/services/pandaReqwest.js

```diff
@@ -10,7 +11,8 @@ export function pandaReqwest(reqwestBody) {
         })
         .fail(err => {
           if (err.status == 419) {
-            const reauthUrl = '/reauth';
+            const store = getStore();
+            var reauthUrl = store.getState().config.reauthUrl;
```

Comment:
What happens if config has been set to false in the reducer or the config isn't there how does it fail?